bin/rename

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+CURRENT_NAME="PhoenixStarter"
+CURRENT_OTP="phoenix_starter"
+
+echo "Please enter new name (current: $CURRENT_NAME)"
```

Comment:
put $CURRENT_NAME and any variable inside double quotes to prevent globbing and word splitting